src/payment/strategies/moneris/moneris-payment-strategy.ts

```diff
@@ -127,10 +127,20 @@ export default class MonerisPaymentStrategy implements PaymentStrategy {
         const cssTextboxExpiry = style?.cssTextboxExpiry ?? 'width:40px;';
         const csstexboxCvd = style?.csstexboxCvd ?? 'width:40px';
 
+        const monerisLabels: MonerisHostedFieldsLabels = {
+            pan_label: initializationData?.creditCardLabel || 'Credit Card Number',
+            exp_label: initializationData?.expiryDateLabel || 'Expiration',
+            cvd_label: initializationData?.cvdLabel || 'CVD',
```

Comment:
Should these only be optional as options? Based on how the query param was written, I would assume they are all required in order to generate the query param. Otherwise we could end up with things like `pan_label=undefined`.

If so, we can force all the previously optional values to be mandatory by wrapping `MonerisHostedFieldsLabels` in `Required`: `Required<MonerisHostedFieldsLabels>` so no value can be `undefined`.